service/src/test/java/bio/terra/catalog/datarepo/DatarepoServiceTest.java

```diff
@@ -54,13 +55,22 @@ void beforeEach() {
     doReturn(unauthenticatedApi).when(datarepoService).unauthenticatedApi();
   }
 
+  @Test
+  void accessMapping() throws Exception {
+    assertThat(DatarepoService.SNAPSHOT_ROLE_TO_DATASET_ACCESS_LEVEL.get(DatarepoService.STEWARD_ROLE_NAME), is(DatasetAccessLevel.OWNER));
```

Comment:
I don't think we should have tests that only test static data structures, as they duplicate the state of the data in test form only. Instead we should have a test on an operation that depends on this object.